test/mpi/configure.ac

```diff
@@ -741,14 +734,17 @@ PAC_POP_FLAG([LDFLAGS])
 PAC_POP_FLAG([LIBS])
 
 # setup which tests "make testing" will run
-TESTLIST=testlist,testlist.cvar
+TESTLIST=testlist
 TESTDIRS=
 if test $have_gpu = "yes" ; then
     TESTLIST="${TESTLIST},testlist.gpu"
 fi
 if test $enable_dtpools = "yes" ; then
     TESTLIST="$TESTLIST},testlist.dtp"
 fi
+if test $enable_collalgo_tests = "yes" ; then
```

Comment:
Since it enables `testlist.cvar`, should we name the option `--enable-cvar-tests`?

Reply:
Not a bad idea, though I worry it might be misleading given other CVAR-based tests we have in the suite. Not to mention tests that use the `MPI_T_cvar` interfaces 😅. Maybe we change the name of the generated testlist to `testlist.collalgo` or `testlist.ext`?

Reply:
+1 to `testlist.collalgo`.

Reply:
done